server/resource_server/service/service.go

```diff
@@ -177,15 +172,15 @@ func (s *service) ProfileClient(userName string) (models.ClientResponseOutput, e
 	return clientModel, nil
 }
 
-func (s *service) VerifyEmail(userID uint) error {
+func (s *service) VerifyEmail(userID uint, userEmail string) error {
```

Comment:
Hmmmmm, there should definitely be a Mandelbrot Set emoji that I can insert here...